swiftly/client/standardclient.py

```diff
@@ -514,19 +516,19 @@ def request(self, method, path, contents, headers, decode_json=False,
                         method, conn_path + path, contents, titled_headers)
                 else:
                     conn.putrequest(method, conn_path + path)
-                    content_length = None
```

Comment:
> Does Python 3 have trouble with Nones in certain cases?

Comparisons are made between int and None later, which is [not valid](https://docs.python.org/3.1/whatsnew/3.0.html#ordering-comparisons).

> This change seems wrong

Definitely incomplete, yes.

I'm curious, why did you write this instead of using urllib/requests? The whole method could be replaced with one or two lines.